pkg/verify/verify.go

```diff
@@ -348,11 +255,20 @@ func (v *ReleaseVerifier) cacheVerification(releaseDigest string, signedWith [][
 	v.signatureCache[releaseDigest] = signedWith
 }
 
-// checkFileSignatures reads signatures as "signature-1", "signature-2", etc out of a directory until
-// either the provided fn returns an error, false, or no such file exists. No more than maxSignaturesToCheck
-// will be read.
-func checkFileSignatures(ctx context.Context, dir string, maxSignaturesToCheck int, fn func(path string, signature []byte) (bool, error)) error {
-	base := filepath.Join(dir, "signature-")
+type fileStore struct {
+	directory string
+}
+
+// Signatures reads signatures as "signature-1", "signature-2", etc. out of a directory.
```

Comment:
`// Signatures reads signatures as "signature-1", "signature-2", etc. out of digest named sub-directory of a directory.`
or some such

Reply:
Reworded to:

```go
// Signatures reads signatures as "signature-1", "signature-2", etc. out of a digest-based subdirectory.
```

with 487d860 -> 6044c9c.